### DISCUSSION TITLE
docx: how to have carriage return in a cell table?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#6069] [Q&A]
### Description

Hi,
if I use this qmd

````qmd
---
title: "Title"
format:
  docx:
    toc: false
    number-sections: false
    highlight-style: github
---

# A title

Lorem ipsum

|a|b|
|---|---|
|1|2<br>3|

````

the docx output does not apply a carriage return where I have the `<br>` tag.

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/29272d1d-8f7e-44ff-aefe-06c8f328b611)

Is the "[Grid tables](https://quarto.org/docs/authoring/tables.html#grid-tables)", the only way to have a carriage return in the docx table output?

If yes, do you know some cli tool to convert json or csv to "Grid tables"?

Thank you

**Comment by mcanouil** [comment]
Pipe table, the one you are using, is not able to be read by Pandoc for multiline cells, see <https://pandoc.org/MANUAL.html#tables> for details about this.

So yes, you need something else than pipe table, you can use HTML table as described (now) in <https://quarto.org/docs/authoring/tables.html#html-tables>.

Alternatively, you can use `pander` in R, see an example below:

````qmd
---
format: markdown
---

```{r}
library(pander)

# Define a Markdown pipe table
md_table <- "| Column 1 | Column 2 | Column 3 |\n| --- | --- | --- |\n| A | B | C |\n| D | E | F |\n"

# Convert the Markdown pipe table to a data frame
df <- read.table(text = md_table, sep = "|", header = TRUE, stringsAsFactors = FALSE)
df <- df[, -c(1, ncol(df))]

# Convert the data frame to a grid table using pander
pander(df, style = "grid")
```
````
